nginx/assets/docs/spec.yaml

```diff
@@ -0,0 +1,259 @@
+name: Nginx
+files:
+  - name: README.md
+    sections:
+      - template: overview
+        overrides:
+          description: |
+            The Datadog Agent can collect many metrics from NGINX instances, including (but not limited to)::
+
+            - Total requests
+            - Connections (e.g. accepted, handled, active)
+
+            For users of NGINX Plus, the commercial version of NGINX, the Agent can collect the significantly more metrics that NGINX Plus provides, like:
+
+            - Errors (e.g. 4xx codes, 5xx codes)
+            - Upstream servers (e.g. active connections, 5xx codes, health checks, etc.)
+            - Caches (e.g. size, hits, misses, etc.)
+            - SSL (e.g. handshakes, failed handshakes, etc.)
+      - template: setup
+      - template: setup/installation
+        overrides:
+          description: |
+            The NGINX check pulls metrics from a local NGINX status endpoint, so your `nginx` binaries need to have been compiled with one of two NGINX status modules:
+
+            - [stub status module][2] - for open source NGINX
+            - [http status module][3] - only for NGINX Plus
+            [2]: https://nginx.org/en/docs/http/ngx_http_stub_status_module.html
+            [3]: https://nginx.org/en/docs/http/ngx_http_status_module.html
```

Comment:
You *should* be able to number these independent of other sections, i.e. start counting at `1`.